{
  try {
    logBasic(Messages.getString("RowsToResult.Log.StartingToRun"));
    while (processRow(meta,data) && !isStopped())     ;
  }
 catch (  Exception e) {
    logError(Messages.getString("RowsToResult.Log.UnexpectedError") + " : " + e.toString());
    logError(Const.getStackTracker(e));
    setErrors(1);
    stopAll();
  }
 finally {
    dispose(meta,data);
    logSummary();
    markStop();
  }
}
